app/views/favorites/of_user.ctp

```diff
@@ -43,45 +55,56 @@ $numberOfSentences = count($favorites);
-    if ($numberOfSentences > 0) {
-        $type = 'mainSentence';
-        $parentId = null;
-        $withAudio = false;
-        $ownerName = null;
-        foreach ($favorites as $favorite) {
-            if (empty($favorite['Sentence']['text'])) {
-                $sentenceId = $favorite['Favorite']['favorite_id'];
-                $linkToSentence = $html->link(
-                    '#'.$sentenceId,
-                    array(
-                        'controller' => 'sentences',
-                        'action' => 'show',
-                        $sentenceId
-                    )
-                );
+    if ($userExists === false) {
+        $commonModules->displayNoSuchUser($username, $backLink);
+    } else {
+        echo '<h2>';
+        echo $title . ' ('. $numberOfSentences.')';
+        echo '</h2>';
+        if ($numberOfSentences > 0) {
+
+            $paginationUrl = array($username);
+            $pagination->display($paginationUrl);
+
+            $type = 'mainSentence';
+            $parentId = null;
+            $withAudio = false;
+            $ownerName = null;
+            foreach ($favorites as $favorite) {
+                if (empty($favorite['Sentence']['text'])) {
+                    $sentenceId = $favorite['Favorite']['favorite_id'];
+                    $linkToSentence = $html->link(
+                        '#'.$sentenceId,
+                        array(
+                            'controller' => 'sentences',
+                            'action' => 'show',
+                            $sentenceId
+                        )
+                    );
 
-                echo $html->div('sentence deleted',
-                    format(
-                        __('Sentence {id} has been deleted.', true),
-                        array('id' => $linkToSentence)
-                    )
-                );
-            } else {
-                $sentences->displayGenericSentence(
-                    $favorite['Sentence'],
-                    $favorite['Sentence']['Transcription'],
-                    $type,
-                    $parentId,
-                    $withAudio
-                );
+                    echo $html->div('sentence deleted',
+                        format(
+                            __('Sentence {id} has been deleted.', true),
+                            array('id' => $linkToSentence)
+                        )
+                    );
+                } else {
+                    $sentences->displayGenericSentence(
+                        $favorite['Sentence'],
+                        $favorite['Sentence']['Transcription'],
+                        $type,
+                        $parentId,
+                        $withAudio
+                    );
+                }
             }
+            $pagination->display($paginationUrl);
+
+        } else {
+            __('This user does not have any favorites.');
+            echo $html->link(__('Go back to previous page', true), $backLink);
```

Comment:
Yep, I agree it's not useful here. No need to try to keep it consistent with the other pages that have a back link. It's better if the favorites page doesn't have a back link.

For the back links in the other pages, we can remove them later. It's indeed not so urgent, nor so important. But yea, if you are in the mood for cleaning up old code, feel free to make another pull request.